.github/workflows/spell-check.yml

```diff
@@ -0,0 +1,36 @@
+name: Spell check
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+  push:
+    branches:
+    - master
+    - develop
+    on:
+      pull_request:
+        types: [opened, synchronize, reopened]
+
+      push:
+        branches:
+        - master
+        - develop
+
```

Comment:
No need for lines 11-18.

This will cover all PRs:

```
on:
  pull_request:
    types: [opened, synchronize, reopened]
```

This will do `aftermerge` checks for "important" branches:

```
push:
    branches:
    - master
    - develop
```

Reply:
fixed